protocol/messenger.go

```diff
@@ -657,6 +660,17 @@ func (m *Messenger) Start() (*MessengerResponse, error) {
 		return nil, err
 	}
 
+	if m.config.torrentConfig.Enabled {
+		adminCommunities, err := m.communitiesManager.Created()
+		if err == nil && len(adminCommunities) > 0 {
+			available := m.SubscribeMailserverAvailable()
+			go func() {
+				<-available
+				m.InitHistoryArchiveTasks(adminCommunities)
+			}()
+		}
+	}
+
```

Comment:
Ok, I've found the fix for the failing test.

```suggestion
	if m.config.torrentConfig != nil && m.config.torrentConfig.Enabled {
		adminCommunities, err := m.communitiesManager.Created()
		if err == nil && len(adminCommunities) > 0 {
			available := m.SubscribeMailserverAvailable()
			go func() {
				<-available
				m.InitHistoryArchiveTasks(adminCommunities)
			}()
		}
	}

```

Reply:
Argh... I'm sorry you had to go after this. Thank you!

Reply:
It's ok, these sorts of bugs are very hard to find. When a package uses `recover` it makes discovery so tricky.